### PR TITLE
Social_Media_Sidebar #issue68

### PR DESCRIPTION
**Pull request by Surabhi0910**
Added Social Media Sidebar. And also centred the text.
Before: Screenshot (570) 
After: 
Screenshot (569)

### CODE REVIEW

File: css/style.css [hunk 1]
```diff
@@ -1,5 +1,6 @@
 @import url("https://fonts.googleapis.com/css2?family=Poppins:ital,wght@0,300;0,400;0,500;0,700;0,900;1,900&display=swap");
-
+@import url(https://fonts.googleapis.com/css?family=Open+Sans:400,700);
```
Comment (harikesh409) [inline css/style.css:2]: use the default font itself so that the load is less.

File: css/style.css [hunk 2]
```diff
@@ -553,4 +556,83 @@ hr {
...
+	position: fixed;
+	top: 30%;
+}
+#social-sidebar li:first-child a { border-top-right-radius: 5px; }
+#social-sidebar li:last-child a { border-bottom-right-radius: 5px; }
+ 
+#social-sidebar a {
+	background: rgba(31, 30, 30, 0.7);
+	color: #fff;
+    text-decoration: none;
```
Comment (harikesh409) [inline css/style.css:575]: correct the indentation.

File: index.html [hunk 3]
```diff
@@ -89,6 +89,28 @@
     </header>
 
     <!-- header ends -->
+     <!--social media sidebar starts-->
+     <ul id="social-sidebar">
+        <li>
+          <a class="entypo-twitter"><span>Tweeter</span></a>
```
Comment (harikesh409) [inline index.html:95]: correct the spelling of twitter

**Review by harikesh409 — CHANGES_REQUESTED** [review]

File: css/style.css (hunk 2 above)
Comment (Surabhi0910) [reply css/style.css]: done.

File: index.html (hunk 3 above)
Comment (Surabhi0910) [reply index.html]: Done.

**Comment by Surabhi0910** [comment]
Done.

**Comment by Surabhi0910** [comment]
Done.

File: css/style.css (hunk 1 above)
Comment (Surabhi0910) [reply css/style.css]: Done.

**Review by harikesh409 — APPROVED** [review]

**Pull request merged**